docs/requirements.txt

```
@@ -0,0 +1,12 @@
+sphinx
+sphinx_rtd_theme
```

Let's add this, allows us to make tabs, etc. - super useful :)
```suggestion
sphinx-design
sphinx_rtd_theme
```
https://github.com/ECP-WarpX/WarpX/pull/3361